src/bootstrap/builder.rs

```diff
@@ -487,7 +473,15 @@ impl<'a> ShouldRun<'a> {
         self.paths(&[path])
     }
 
-    // multiple aliases for the same job
+    /// Multiple aliases for the same job.
+    ///
+    /// This differs from [`path`] in that multiple calls to path will end up calling `make_run`
+    /// multiple times, whereas a single call to `paths` will only ever generate a single call to
+    /// `paths`.
```

Comment:
"a single call to paths will only ever generate a single call to paths" eh... what does that mean?^^

Reply:
oh, this is a blast from the past. probably i meant "a single call to `make_run`", by analogy with the docs above? i honestly don't remember though, it's been ages.